packages/url_launcher/ios/Classes/UrlLauncherPlugin.m

```diff
@@ -96,22 +101,23 @@ - (BOOL)canLaunchURL:(NSString *)urlString {
   return [application canOpenURL:url];
 }
 
-- (void)launchURL:(NSString *)urlString result:(FlutterResult)result {
+- (void)launchURL:(NSString *)urlString options:(NSDictionary<UIApplicationOpenExternalURLOptionsKey, id> *)options result:(FlutterResult)result {
   NSURL *url = [NSURL URLWithString:urlString];
   UIApplication *application = [UIApplication sharedApplication];
-  if ([application respondsToSelector:@selector(openURL:options:completionHandler:)]) {
```

Comment:
I'm missing context on the fix that added this check, is this completely equivalent to checking for iOS 10.0 and above?

Reply:
Yes it is equivalent; however, in recent updates xcode starts to give warning when api gets called without a version check, so this update is to suppress the warning